backend/src/test/java/com/woowacourse/levellog/acceptance/AcceptanceTest.java

```diff
@@ -63,5 +88,112 @@ public void setUp(final RestDocumentationContextProvider contextProvider) {
                                         prettyPrint()
                                 )
                 ).build();
+
+        masterToken = getToken(login(MASTER));
+        masterId = getMemberId(login(MASTER));
+    }
+
+    protected ValidatableResponse requestCreateTeam(final String title, final String host,
+                                                    final String... participants) {
+        final List<Long> participantIds = Arrays.stream(participants)
+                .map(this::login)
+                .map(ValidatableResponseOptions::extract)
+                .map(it -> it.as(LoginResponse.class))
+                .map(LoginResponse::getId)
+                .collect(Collectors.toList());
+        final ParticipantIdsRequest participantIdsRequest = new ParticipantIdsRequest(participantIds);
+        final TeamRequest request = new TeamRequest(title, title + "place", LocalDateTime.now(), participantIdsRequest);
+
+        final String token = getToken(login(host));
+
+        return RestAssured.given().log().all()
+                .header(HttpHeaders.AUTHORIZATION, "Bearer " + token)
+                .body(request)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when()
+                .post("/api/teams")
+                .then().log().all();
+    }
+
+    private ValidatableResponse login(final String nickname) {
+        try {
+            final GithubProfileResponse response = new GithubProfileResponse(String.valueOf(
+                    ((int) System.currentTimeMillis())), nickname,
```

Comment:
```suggestion
                    ((int) System.nanoTime())), nickname,
```
뒤에 네트워크 요청이 있어서 시간 내에 두 깃헙 아이디가 생성되진 않겠지만.. nanoTime()을 쓰는게 더 안전해보여요. 🥲